exercises/devops/README.md

```diff
@@ -252,6 +252,33 @@ Reliability, when used in DevOps context, is the ability of a system to recover
 
 <details>
 <summary>How a web server works?</summary><br><b>
+	According to MDN Web Docs
```

Comment:
Move it the beginning of the line + reference to site

Reply:
ok i will do this